Management.Api/Web/components/auth/Login.jsx

```diff
@@ -63,6 +69,11 @@ class Login extends React.Component {
       }, handleError)
       .then((data) => {
         if (data) {
+          if (data.Message && data.Message === "Locked Out") {
+            lockoutError()
+            return
```

Comment:
Put the `return lockoutError()`. Generally you never use empty returns in js, and either way here you'd return `undefined`.